<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>眼鏡推薦</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>600</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_takePict">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>650</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>拍照</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="faceshapeOfUser">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>650</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancelTakePict">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>650</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>取消重拍</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>125</y>
      <width>200</width>
      <height>450</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_revert">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>650</y>
      <width>80</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>回復原狀</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
